### PR TITLE
Adding support for sync_time_with_host flag

### PR DESCRIPTION
**Pull request by dkalleg**

### CODE REVIEW

File: builtin/providers/vsphere/resource_vsphere_virtual_machine.go [hunk 1]
```diff
@@ -1710,6 +1722,16 @@ func (vm *virtualMachine) setupVirtualMachine(c *govmomi.Client) error {
...
 		configSpec.GuestId = "otherLinux64Guest"
 	}
+	if vm.syncTimeWithHost {
+		if configSpec.Tools == nil {
+			toolsConfig := types.ToolsConfigInfo{
+				SyncTimeWithHost: &vm.syncTimeWithHost,
+			}
+			configSpec.Tools = &toolsConfig
+		} else {
+			*configSpec.Tools.SyncTimeWithHost = vm.syncTimeWithHost
```
Comment (dagnello) [inline builtin/providers/vsphere/resource_vsphere_virtual_machine.go]: SyncTimeWithHost might also be null here, should check for that too

Comment (dkalleg) [inline builtin/providers/vsphere/resource_vsphere_virtual_machine.go]: Good call, will push a patch shortly.

**Comment by chrislovecnm** [comment]
Do we need docs and a test??

**Comment by dkalleg** [comment]
@chrislovecnm This is a fairly granular feature and may cause more overhead than its worth to add an additional test for this.  I'm, of course, open to go with the popular decision on this.

File: builtin/providers/vsphere/resource_vsphere_virtual_machine.go [hunk 2]
```diff
@@ -1103,6 +1114,7 @@ func resourceVSphereVirtualMachineRead(d *schema.ResourceData, meta interface{})
 	d.Set("cpu", mvm.Summary.Config.NumCpu)
 	d.Set("datastore", rootDatastore)
 	d.Set("uuid", mvm.Summary.Config.Uuid)
+	d.Set("sync_time_with_host", *mvm.Config.Tools.SyncTimeWithHost)
```
Comment (stack72) [inline builtin/providers/vsphere/resource_vsphere_virtual_machine.go]: This doesn't need deferenced - d.Set() does this in a safe way :)

**Comment by paddycarver** [comment]
Hi! This PR has been open for a while and waiting response from the community without any activity, so I'm going to close it out. We do this to try to keep our attention on issues that are actively impacting the community, it's not a comment on the PR itself. If there is still interest in this PR, please feel free to reply, and we'll re-open the PR. Thanks!

**Pull request closed without merging**